experimental/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ All notable changes to experimental packages in this project will be documented
 ### :bug: (Bug Fix)
 
 * fix(sdk-node): use resource interface instead of concrete class [#3803](https://github.com/open-telemetry/opentelemetry-js/pull/3803) @blumamir
+* fix(exporter-logs-otlp-grpc): change OTLPLogsExporter to OTLPLogExporter [#3819](https://github.com/open-telemetry/opentelemetry-js/pull/3819) @fuaiyi
```

Comment:
This is a breaking change it should be in the breaking section

Reply:
> This is a breaking change it should be in the breaking section

fixed